style/utils.css

```diff
@@ -1,29 +1,50 @@
 /* utils classes */
 
 .flex {
-  display: flex;
-  justify-content: center;
-  align-items: center;
+    display: flex;
+    justify-content: center;
+    align-items: center;
 }
 .f-col {
-  flex-direction: column;
+    flex-direction: column;
 }
 .f-row {
-  flex-direction: row;
+    flex-direction: row;
 }
 .f-justify-start {
-  align-items: flex-start;
+    align-items: flex-start;
 }
 .f-align-start {
-  align-items: flex-start;
+    align-items: flex-start;
 }
 .f-space-btw {
-  justify-content: space-between;
+    justify-content: space-between;
 }
 .f-space-eve {
-  justify-content: space-evenly;
+    justify-content: space-evenly;
 }
 .f-wrap {
-  flex-wrap: wrap;
+    flex-wrap: wrap;
 }
 /* utils classes */
+ul li::after {
```

Comment:
we don't want to add same hover effect on every list, so select navbar one's only
like: `.nav-right ul li::after`